From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Fri, 10 Nov 2023 14:22:04 +0900
Subject: Remove AllCan Read/Write

Refs https://chromium-review.googlesource.com/c/v8/v8/+/5006387
Patch will be removed once upstream lands the relevant v8 roll.

diff --git a/src/js_native_api_v8.cc b/src/js_native_api_v8.cc
index 10b821119d3ff2dd112d6cde195f65a2c34b4a2d..22eac1c65eab1bd632f461fa2fdfb917cc63dc1a 100644
--- a/src/js_native_api_v8.cc
+++ b/src/js_native_api_v8.cc
@@ -928,8 +928,7 @@ napi_define_class(napi_env env,
       tpl->PrototypeTemplate()->SetAccessorProperty(property_name,
                                                     getter_tpl,
                                                     setter_tpl,
-                                                    attributes,
-                                                    v8::AccessControl::DEFAULT);
+                                                    attributes);
     } else if (p->method != nullptr) {
       v8::Local<v8::FunctionTemplate> t;
       STATUS_CALL(v8impl::FunctionCallbackWrapper::NewTemplate(
